src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./search-form.scss"
+import Button from "../Button/Button";
+import { Formik } from 'formik';
+import {useHistory} from 'react-router-dom';
+
+const SearchForm = () => {
+    const history = useHistory();
+
+    return(
+        <div className="search-block">
+            <h1>Find your movie</h1>
+            <Formik
+                initialValues={{ search: "" }}
+                onSubmit={(values) => {
+                    history.push({
+                        pathname: `/search/${values.search}`,
+                    })
+                }}
```

Comment:
how about extracting it to a separate function?

Reply:
fixed